src/VirtualClient/VirtualClient.TestFramework/InMemoryProcess.cs

```diff
@@ -146,6 +146,14 @@ public void Kill()
             this.OnKill?.Invoke();
         }
 
+        /// <summary>
+        /// Kills the fake process, and associated child processes.
+        /// </summary>
+        public void Kill(bool entireProcessTree)
```

Comment:
I would make it into the original method and make it a default. When VC kills threads, we almost always want to kill the entire tree

Reply:
Made it "true" by default in process.Kill(), in ProcessProxy.cs. Thus, process.Kill() is same as process.kill(true). It will be applied to all workloads. In case somewhere false is needed explicitly, the process.Kill(false) or process.SafeKill(false) can be used.